streams/src/main/java/org/apache/kafka/streams/StoreQueryParams.java

```diff
@@ -21,12 +21,11 @@
 import java.util.Objects;
 
 /**
- * Represents all the query options that a user can provide to state what kind of stores it is expecting.
+ * Represents all the query options that a user can provide to state what kind of stores we are expecting.
```

Comment:
cosmetic: `Represents all the query options that a user can provide to control the kind of stores we would like to fetch` ?

Reply:
Done.